paddle/fluid/platform/mkldnn_helper.h

```diff
@@ -533,5 +533,709 @@ inline bool HasOpFLOAT32DataType(const paddle::framework::OpDesc* op) {
+      return "aBdeC48b2c";
+    case MKLDNNMemoryFormat::aBdeC48b4c:
+      return "aBdeC48b4c";
+    case MKLDNNMemoryFormat::aBdec64b:
+      return "aBdec64b";
+    case MKLDNNMemoryFormat::aBdeC64b2c:
+      return "aBdeC64b2c";
+    case MKLDNNMemoryFormat::aBdeC64b4c:
+      return "aBdeC64b4c";
+    case MKLDNNMemoryFormat::adecb:
+      return "adecb";
+    case MKLDNNMemoryFormat::adeCb2c:
+      return "adeCb2c";
+    case MKLDNNMemoryFormat::adeCb4c:
+      return "adeCb4c";
+    case MKLDNNMemoryFormat::Acdeb32a:
+      return "Acdeb32a";
+    case MKLDNNMemoryFormat::AcdeB32a2b:
+      return "AcdeB32a2b";
+    case MKLDNNMemoryFormat::AcdeB32a4b:
+      return "AcdeB32a4b";
+    case MKLDNNMemoryFormat::Acdeb48a:
+      return "Acdeb48a";
+    case MKLDNNMemoryFormat::AcdeB48a2b:
+      return "AcdeB48a2b";
+    case MKLDNNMemoryFormat::AcdeB48a4b:
+      return "AcdeB48a4b";
+    case MKLDNNMemoryFormat::Acdeb64a:
+      return "Acdeb64a";
+    case MKLDNNMemoryFormat::AcdeB64a2b:
+      return "AcdeB64a2b";
+    case MKLDNNMemoryFormat::AcdeB64a4b:
+      return "AcdeB64a4b";
+    case MKLDNNMemoryFormat::cdeBa2b:
+      return "cdeBa2b";
+    case MKLDNNMemoryFormat::cdeBa4b:
+      return "cdeBa4b";
+    case MKLDNNMemoryFormat::aBdefc32b:
+      return "aBdefc32b";
+    case MKLDNNMemoryFormat::aBdefC32b2c:
+      return "aBdefC32b2c";
+    case MKLDNNMemoryFormat::aBdefC32b4c:
+      return "aBdefC32b4c";
+    case MKLDNNMemoryFormat::aBdefc48b:
+      return "aBdefc48b";
+    case MKLDNNMemoryFormat::aBdefC48b2c:
+      return "aBdefC48b2c";
+    case MKLDNNMemoryFormat::aBdefC48b4c:
+      return "aBdefC48b4c";
+    case MKLDNNMemoryFormat::aBdefc64b:
+      return "aBdefc64b";
+    case MKLDNNMemoryFormat::aBdefC64b2c:
+      return "aBdefC64b2c";
+    case MKLDNNMemoryFormat::aBdefC64b4c:
+      return "aBdefC64b4c";
+    case MKLDNNMemoryFormat::adefcb:
+      return "adefcb";
+    case MKLDNNMemoryFormat::adefCb2c:
+      return "adefCb2c";
+    case MKLDNNMemoryFormat::adefCb4c:
+      return "adefCb4c";
+    case MKLDNNMemoryFormat::format_tag_last:
+      return "format_tag_last";
+    case MKLDNNMemoryFormat::NCdhw32n32c:
+      return "NCdhw32n32c";
+    case MKLDNNMemoryFormat::OwI16o2i:
+      return "OwI16o2i";
+    case MKLDNNMemoryFormat::OdhwI16o2i:
+      return "OdhwI16o2i";
+    case MKLDNNMemoryFormat::gOwI16o2i:
+      return "gOwI16o2i";
+    case MKLDNNMemoryFormat::gOhwI16o2i:
+      return "gOhwI16o2i";
+    case MKLDNNMemoryFormat::OIw4o8i8o4i:
+      return "OIw4o8i8o4i";
+    case MKLDNNMemoryFormat::OIdhw4o8i8o4i:
+      return "OIdhw4o8i8o4i";
+    case MKLDNNMemoryFormat::gOIw4o8i8o4i:
+      return "gOIw4o8i8o4i";
+    case MKLDNNMemoryFormat::gOIdhw4o8i8o4i:
+      return "gOIdhw4o8i8o4i";
+    case MKLDNNMemoryFormat::gOdhwI16o2i:
+      return "gOdhwI16o2i";
+    case MKLDNNMemoryFormat::OhwI16o4i:
+      return "OhwI16o4i";
+  }
+  return "unknown";
```

Comment:
This option is necessary because if oneDNN would add a new format tag compilation would result in a failure